NAMESPACE

```diff
@@ -41,7 +41,9 @@ S3method(unite,SingleCellExperiment)
 S3method(unnest,tidySingleCellExperiment_nested)
 export("%>%")
 export(add_count)
+export(aggregate_cells)
 export(arrange)
+export(as_matrix)
```

Comment:
the only exported function should be `aggregate_cells`? Or am I missing something?

Reply:
Apologies, as_matrix has now been removed from the exported function.